django/contrib/postgres/aggregates/base.py

```diff
@@ -0,0 +1,8 @@
+from django.db.models import Aggregate
+
+
+class DistinctAggregate(Aggregate):
```

Comment:
I wonder if this is something we could define in `django.db.models.Aggregate` and have `Count` use it as well? What do you think @orf?